Source/BSGOutOfMemoryWatchdog.m

```diff
@@ -155,7 +155,7 @@ - (BOOL)computeDidOOMLastLaunchWithConfig:(BugsnagConfiguration *)config {
             NSString *osVersion = [BSG_KSSystemInfo osBuildVersion];
             NSDictionary *appInfo = [[NSBundle mainBundle] infoDictionary];
             NSString *appVersion =
-                [appInfo valueForKey:(__bridge NSString *)kCFBundleVersionKey];
+                [appInfo valueForKey:@BSG_KSSystemField_BundleShortVersion];
```

Comment:
If this is being used to detect if the app has the same version and hence report the OOM, should this be on app version and bundle version rather than just app version?

Reply:
That's a good question. The safest thing is to check both, so I'll make it do that.